requirements.txt

```diff
@@ -1,6 +1,33 @@
-matplotlib==3.0.2
-PySide==1.2.4
-# PySide2==5.12.0
-Shiboken==1.2.2
-six==1.12.0
-Markdown==3.2.2
+area==1.1.1
+bzr_pqm==1.4.0
+cog==0.6.1
+ConfigParser==5.3.0
+defusedxml==0.7.1
+GitPython==3.1.31
+ifcopenshell==0.7.0.230318
+kdtree==0.16
+ladybug==0.0.2
+matplotlib==3.6.3
+numpy==1.24.2
+ocl==0.12
+opencamlib==2023.1.11
```

Comment:
aren't ocl and opencamlib the same thing?

Reply:
Yes, it looks like this error comes from this
```
# OCL must be installed
try:
    try:
        import ocl
    except ImportError:
        import opencamlib as ocl
except ImportError:
```
where the tool marks both as a requirement. Which one should be kept?